ember-bootstrap/addon/components/bs-modal/header.hbs

```diff
@@ -1,20 +1,15 @@
-{{! @glint-nocheck }}
 {{#let
   (ensure-safe-component (bs-default @titleComponent (component "bs-modal/header/title")))
   (component (ensure-safe-component (bs-default @closeComponent (component "bs-modal/header/close"))) onClick=@onClose)
 as |Title Close|
 }}
   <div class="modal-header" ...attributes>
     {{#if (has-block-params)}}
-      {{yield
-        (hash
-          title=Title
-          close=Close
-        )
+      {{yield (hash title=Title close=Close)
       }}
     {{else}}
       {{#if (has-block)}}
-        {{yield}}
+        {{yield (hash title=Title close=Close)}}
```

Comment:
I don't think we need to yield here. We are checking before of the block should receive parameters.

If Glimmer does not support that well, I think we should just remove the `has-block-params` code path. Simplifying the implementation may be good anyways. At least I don't see benefits in conditionally yuelding the component. I doubt that Embroider is able to detect it and tree-shake the child component if it's not needed.